src/commcare_cloud/ansible/roles/commcarehq/templates/supervisor_pillowtop.conf.j2

```diff
@@ -2,7 +2,8 @@
 {% for num_process in range(params.get('start_process', 0), params.get('start_process', 0) + params.get('num_processes', 1)) %}
 
 [program:commcare-hq-{{ deploy_env }}-pillowtop-{{ pillow_name }}-{{ num_process }}]
-environment=TMPDIR="{{ encrypted_tmp }}",READ_FROM_PLPROXY_STANDBYS="1"
+environment=TMPDIR="{{ encrypted_tmp }}",READ_FROM_PLPROXY_STANDBYS="1" {% if prometheus_monitoring_enabled|default(False) %},prometheus_multiproc_dir="{{ cchq_home}}/www/{{ deploy_env }}/metrics" {% endif %}
```

Comment:
what do you think about re-working this to use the same style as the celery / webworker supervisor files?

Reply:
Updated